src/app/header/components/search-bar/search-bar.module.ts

```diff
@@ -1,10 +1,11 @@
 import { NgModule } from "@angular/core";
 import { ReactiveFormsModule } from "@angular/forms";
+import { CommonModule } from "@angular/common";
 
 import { SearchBarComponent } from "./search-bar.component";
 
 @NgModule({
-  imports: [ReactiveFormsModule],
+  imports: [CommonModule, ReactiveFormsModule],
```

Comment:
Why did you need to add this depencency here?

Reply:
Actually I don't 😅 I think it was something that I was trying in the component and I forgot to remove it. Thank you!

Reply:
<img width="316" alt="image" src="https://user-images.githubusercontent.com/104014855/182128071-aaea0890-b705-43c3-8a5b-112053f735c8.png">

Well, I do need it hahaha

Reply:
Yes you did!